app/gui2/src/components/GraphNode.vue

```diff
@@ -414,12 +417,35 @@ const dragPointer = usePointer((pos, event, type) => {
   }
 })
 
+const suggestionDbStore = useSuggestionDbStore()
+
 const expressionInfo = computed(() =>
   projectStore.computedValueRegistry.getExpressionInfo(props.node.rootSpan.id),
 )
 const outputTypeName = computed(() => expressionInfo.value?.typename ?? 'Unknown')
 const executionState = computed(() => expressionInfo.value?.payload.type ?? 'Unknown')
+const suggestionEntry = computed(() => {
+  const method = expressionInfo.value?.methodCall?.methodPointer
+  if (method == null) return undefined
+  const filtering = new Filtering({
+    pattern: method.name,
+    qualifiedNamePattern: method.module,
+    selfType: method.definedOnType as QualifiedName,
+    showLocal: true,
+    showUnstable: true,
+  })
+  if (filtering == null) return undefined
```

Comment:
When filtering could be null? Don't constructor always return an object?

Reply:
nice catch, thanks